mkdocs.yml

```
@@ -2,8 +2,9 @@ repo_url: https://github.com/cirruslabs/cirrus-ci-docs/
 site_url: https://cirrus-ci.org
 edit_uri: ""
 
-site_name: Cirrus CI Documentation
+site_name: Cirrus CI
 site_author: Cirrus Labs
+copyright: © Cirrus Labs 2019
```

```suggestion
copyright: © Cirrus Labs 2017-present
```
no need to update anymore